test/e2e/warnings_test.go

```diff
@@ -72,27 +81,62 @@ spec:
   image: my-awesome-cron-image
 `
 
+	expectedOutputTemplate := `
```

Comment:
lets just duplicate this template for each section, instead of doing bunch of replaces. assertion will become more readable imho.

Reply:
Noted! will do